### PR TITLE
Test changes

### PR DESCRIPTION
**Pull request by ahsu1230**
Sorry for the confusion & delay (had trouble with branching and stuffs)

Adding an integration test (site-keeps-export_spec.js)
Adding a database test
which addresses some small changes (variable naming, subtitle for UI, removing unused variables, etc.)

@martinraison @stkem @eishay

### CODE REVIEW

**Comment by martinraison** [comment]
cool! good to merge

**Comment by stkem** [comment]
:+1:

File: kifi-backend/modules/shoebox/test/com/keepit/controllers/website/KeepsControllerTest.scala [hunk 1]
```diff
@@ -760,7 +760,61 @@ class KeepsControllerTest extends Specification with ApplicationInjector {
       }
     }
 
-    "keepExports" in  {
+    "export keeps" in {
+      running(new ShoeboxApplication(controllerTestModules: _*)) {
```
Comment (eishay) [inline kifi-backend/modules/shoebox/test/com/keepit/controllers/website/KeepsControllerTest.scala]: @martinraison @stkem isn't it better to use 

``` scala
withDb(myGuiceModules) { implicit injector =>
   ...
```

In this case?
The `val userRepo = inject[UserRepo]` etc should come free in this case.

Comment (martinraison) [inline kifi-backend/modules/shoebox/test/com/keepit/controllers/website/KeepsControllerTest.scala]: Good point - I don't think we should need an application here.

**Comment by eishay** [comment]
Thanks!

**Pull request merged**